### ISSUE TITLE
error: Undefined index: zendframework/zend-i18n when select Zend View

### ISSUE THREAD

**Issue by samsonasik** [zendframework/zend-expressive-skeleton#4]
The error like the following:
```
  Which template engine you want to use?
  [1] Plates
  [2] Twig
  [3] Zend View installs Zend ServiceManager
  [n] None of the above
  Make your selection or type a composer package name and version (n): 3
  - Adding package zendframework/zend-expressive-zendviewrenderer (^0.1)
Script App\Composer\OptionalPackages::install handling the pre-install-cmd event terminated with an exception


                                            
  [ErrorException]                          
  Undefined index: zendframework/zend-i18n  
```

**Comment by samsonasik** [comment]
The full steps to get  error:
```
$ composer create-project zendframework/zend-expressive-skeleton
Installing zendframework/zend-expressive-skeleton (0.5.0)
  - Installing zendframework/zend-expressive-skeleton (0.5.0)
    Downloading: 100%         

Created project in /Users/samsonasik/www/zend-expressive-skeleton
> App\Composer\OptionalPackages::install
Setup data and cache dir
Setting up optional packages

  Which router you want to use?
  [1] Aura.Router
  [2] FastRoute
  [3] Zend Router
  Make your selection or type a composer package name and version (FastRoute): 3
  - Adding package zendframework/zend-expressive-zendrouter (^0.1)
  - Copying /config/autoload/routes.global.php

  Which container you want to use for dependency injection?
  [1] Aura.Di
  [2] Pimple-interop
  [3] Zend ServiceManager
  Make your selection or type a composer package name and version (Zend ServiceManager): 3
  - Adding package zendframework/zend-servicemanager (^2.5)
  - Adding package ocramius/proxy-manager (^1.0)
  - Copying /config/container.php

  Which template engine you want to use?
  [1] Plates
  [2] Twig
  [3] Zend View installs Zend ServiceManager
  [n] None of the above
  Make your selection or type a composer package name and version (n): 3
  - Adding package zendframework/zend-expressive-zendviewrenderer (^0.1)
Script App\Composer\OptionalPackages::install handling the pre-install-cmd event terminated with an exception


                                            
  [ErrorException]                          
  Undefined index: zendframework/zend-i18n  
                                            


create-project [-s|--stability="..."] [--prefer-source] [--prefer-dist] [--repository-url="..."] [--dev] [--no-dev] [--no-plugins] [--no-custom-installers] [--no-scripts] [--no-progress] [--keep-vcs] [--no-install] [--ignore-platform-reqs] [package] [directory] [version]
```

**Comment by samsonasik** [comment]
I've created PR #5 for it

**Comment by xtreamwayz** [comment]
Since zend view renderer is a stand alone package now, I think zendframework/zend-i18n  should be inside zend-expressive-zendviewrenderer. It is easier to manage extra dependencies there in it's composer.json file. In case zend-view gets an update you don't need to update the skeleton, only the zendviewrenderer package.

**Comment by samsonasik** [comment]
@xtreamwayz ok, I created alternative fixes in https://github.com/zendframework/zend-expressive-zendviewrenderer/pull/1

**Comment by samsonasik** [comment]
the alternative fixes need both https://github.com/zendframework/zend-expressive-skeleton/pull/6 and https://github.com/zendframework/zend-expressive-zendviewrenderer/pull/1 to be merged ;)

**Comment by weierophinney** [comment]
Fixed with #6.
